sentry/sentry.py

```diff
@@ -46,22 +54,57 @@ def cog_unload(self):
         self.bot.remove_before_invoke_hook(self.before_invoke)
         return super().cog_unload()
 
+    @checks.mod()
+    @commands.group(name="sentry", pass_context=True)
+    async def _sentry(self, ctx: commands.context.Context):
+        """Automatically reply to messages matching certain trigger phrases"""
+
+    @_sentry.command(name="set_env")
+    async def sentry_set_env(self, context: commands.context.Context, new_value: str):
+        """Set sentry environment"""
+        if not context.guild:
+            await context.send("That command is not available in DMs. Sentry environment is a per-guild setting.")
+            return
```

Comment:
You can use the `commands.guild_only` decorator to achieve this exact behaviour automatically. See the notes cog as an example of this.